vector/build.gradle

```
@@ -443,7 +443,7 @@ dependencies {
 
     implementation libs.github.glide
     kapt libs.github.glideCompiler
-    implementation 'com.github.yalantis:ucrop:2.2.7'
+    implementation 'com.github.yalantis:ucrop:2.2.8-native'
```

Keep using the non-native version

```suggestion
    implementation 'com.github.yalantis:ucrop:2.2.8'
```